web/services/firestore/index.js

```diff
@@ -0,0 +1,15 @@
+export const constructFirestoreObject = (firestore, key) => {
+  const { data } = firestore;
+
+  const isLoaded = !!data[key];
```

Comment:
`!!`? also isn't this function retrieving data? why not call it `getFromFirestore()`?

Reply:
yaeh thats what i thought it does, pretty neat dont think ive seen it yet

mmm I think `getFromFirestore(store, key)` is sufficiently self-documenting

Reply:
but its up to you